manifests/server.pp

```diff
@@ -92,15 +92,21 @@
     },
 
     exec { 'pds-migrations':
-      unless  => '/opt/puppetlabs/sbin/pds-ctl rake db:migrate:status',
```

Comment:
We still need something here for the 'unless', to make the exec idempotent and noop compatible (needs to NOT do anything when run in noop)

Reply:
@reidmv I added back the `unless` condition.

I tested it locally and this is the message that it will throw if the first migration is needed:

`Schema migrations table does not exist yet.`